src/common/config_opts.h

```diff
@@ -1019,6 +1019,19 @@ OPTION(bluestore_compression_mode, OPT_STR, "none")  // force|aggressive|passive
 OPTION(bluestore_compression_algorithm, OPT_STR, "snappy")
 OPTION(bluestore_compression_min_blob_size, OPT_U32, 128*1024)
 OPTION(bluestore_compression_max_blob_size, OPT_U32, 512*1024)
+/*
+ * Specifies minimum expected amount of saved allocation units
+ * per single blob to enable compressed blobs garbage collection
+ * 
+ */
+OPTION(bluestore_gc_enable_blob_threshold, OPT_INT, 0)  
+/*
+ * Specifies minimum expected amount of saved allocation units
+ * per all blobsb to enable compressed blobs garbage collection
+ * 
+ */
+OPTION(bluestore_gc_enable_total_threshold, OPT_INT, 0)  
```

Comment:
My considerations on that:
1) There is no need to store data compressed if this provides no storage saving.
2) This way we'll have data layout more consistent with the original write handling. No saving due to compression - no compressed blobs written.
Any other suggestions?